pkg/logs/client/http/destination.go

```diff
@@ -89,8 +90,8 @@ func NewDestination(endpoint config.Endpoint,
 	destinationsContext *client.DestinationsContext,
 	maxConcurrentBackgroundSends int,
 	shouldRetry bool,
-	telemetryName string) *Destination {
-
+	telemetryName string,
+) *Destination {
```

Comment:
Would it be possible to avoid reformatting the code you did not change?

Reply:
Most of the changes are coming from my editor's formatting. These changes did seem reasonable, so I left it. Let me know if they seem too concerning.

Reply:
`git add -p` would help as you can choose which hunks to include in the commit

Reply:
That is good to know. I wasn't aware of it. Most of these changes adhere to good practices and seem reasonable. Are we good to leave them, or should I revert them?

Reply:
> Let me know if they seem too concerning.

Changes like this make reviewers review more code, and it becomes harder to find actual changes when using `git log` and `git blame` later. And we don't all use the same editor and editor settings. If everyone reformats everything to suit their style every time they touch a file, our git history would become much less useful.

If possible, it is better to configure your editor to leave unmodified code as is, or, as @mellon85 mentioned already, be selective in what you include in the PR.

Reply:
Thanks. I have updated the PR by removing the un-needed changes.